src/sms-verification/templates/user_management/sms-verification/verify_form.html

```diff
@@ -0,0 +1,25 @@
+<div class="alert alert-success" ng-show="verified">
+    <p translate>Verification successful.</p>
+</div>
+<div class="alert alert-success" ng-show="resent">
+    <p translate>Activation code resent.</p>
+</div>
+
+<div class="form-group" ng-class="{'has-error': errors.phone_number}">
+    <label for="phone_number" translate>Phone number</label>
+    <input type="phone_number" ng-model="data.phone_number" class="form-control" id="phone_number">
+    <div class="help-block" ng-if="errors.phone_number" ng-bind="errors.phone_number"></div>
+</div>
+<div class="form-group" ng-class="{'has-error': errors.activation_code}">
+    <label for="activation_code" translate>Activation code</label>
+    <input type="activation_code" ng-model="data.activation_code" class="form-control" id="activation_code">
```

Comment:
Ah here again we have `type="activation_code"`. This seems like it should be `type="text"`, or is this doing something special with angular